packages/vault-sdk/README.md

```
@@ -0,0 +1,134 @@
+        break;
+    }
+  },
+);
+
+// Stop later
+sub.stop();
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)